src/Network/Network/help/Set-AzRouteConfig.md

```diff
@@ -64,6 +64,15 @@ This command gets the route table named RouteTable01 by using the Get-AzRouteTab
 The command passes that table to the current cmdlet by using the pipeline operator.
 The current cmdlet modifies the route named Route02, and then passes the result to the **Set-AzRouteTable** cmdlet, which updates the table to reflect your changes.
 
+### Example 2
+
+Updates a route configuration for a route table. (autogenerated)
+
+<!-- Aladdin Generated Example -->
+```powershell
+Set-AzRouteConfig -AddressPrefix <String> -Name 'cert01' -NextHopIpAddress '12.0.0.5' -NextHopType 'VnetLocal'
+```
```

Comment:
This example is missing the required parameter `-RouteTable` that can also be passed through piping.